package/firmware/linux-firmware/qca_ath11k.mk

```diff
@@ -14,7 +22,7 @@ define Package/ath11k-firmware-wcn6750/install
 	$(INSTALL_DATA) \
 		$(PKG_BUILD_DIR)/ath11k/WCN6750/hw1.0/Notice.txt $(1)/lib/firmware/ath11k/WCN6750/hw1.0/
 	$(INSTALL_DATA) \
-	        $(PKG_BUILD_DIR)/ath11k/WCN6750/hw1.0/sc7280/wpss.mbn $(1)/lib/firmware/ath11k/WCN6750/hw1.0/
+		$(PKG_BUILD_DIR)/ath11k/WCN6750/hw1.0/sc7280/wpss.mbn $(1)/lib/firmware/ath11k/WCN6750/hw1.0/
```

Comment:
Is this intentional or probably spaces to tabs fixup?

Reply:
Yes, see also: https://github.com/openwrt/openwrt/commit/24c8f06807181a5fb3a63a74e5ee73d68c6d59f0